### PR TITLE
Create Tag component

### PR DESCRIPTION
**Pull request by RamProg**
## 🚪 Why?

We need to update the looks of the team and employee badges.

## 🔑 What?

Create the Tag component to use in the Details sidebar.

Keep in mind I intentionally don't change the avatar looks for now.

Colors will be updated on a future PR.

## 🏡 Context

image
image

image

image

### CODE REVIEW

File: lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx [hunk 1]
```diff
@@ -17,8 +17,8 @@ const weekdays = (
   />
 )
 const teams = (
-  <BadgesList
-    badges={[
+  <TagsList
+    tags={[
       { text: "Management", avatar: { alt: "M" } },
       { text: "Engineering", avatar: { alt: "E" } },
       { text: "Managers", avatar: { alt: "M" } },
```
Comment (sauldom102) [inline lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx:24]: Why not just taking the first letter of the `text` prop? I see `avatar->alt` kind of redundant

Comment (RamProg) [reply lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx:24]: I think I remember cases where they took the first two letters? If that doesn't happen they yeah, I agree

**Review by sauldom102 — COMMENTED** [review]
In case we have an employee full name we should probably show two initials in the avatar, so in my case it would be "SD"

File: lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx (hunk 1 above)
Comment (nlopin) [reply lib/experimental/Layouts/Utils/DetailsItemsList/index.stories.tsx:24]: maybe take the first one if `alt` is not there to give a meaningful default

File: lib/experimental/Layouts/Utils/Tag/index.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+}
+
+export const Tag = forwardRef<HTMLDivElement, TagProps>(
+  ({ text, avatar }, ref) => {
+    return (
+      <div
+        ref={ref}
+        className="flex flex-row items-center justify-start rounded-md border border-solid border-f1-border py-1 pl-1 pr-2 font-medium text-f1-foreground"
+      >
+        <span className="mr-1.5">
```
Comment (nlopin) [inline lib/experimental/Layouts/Utils/Tag/index.tsx:16]: the parent div is flex, why not using `gap-1.5`?

**Review by nlopin — APPROVED** [review]

**Review by sauldom102 — COMMENTED** [review]
Shall we add an optional `onClick` prop to make them clickable in case we want to add some navigation with it later on? 👀

**Review by sauldom102 — APPROVED** [review]

**Pull request merged**